docs/source/resources/frequently_asked_questions.ipynb

```diff
@@ -810,6 +814,35 @@
     "                                                        'cum_mean'])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The second way is to insert the dataframe into a dictionary mapping its name to a tuple containing specific dataframe information. We then pass in that dictionary to the `dataframes` argument in DFS.\n",
+    "\n",
+    "In this scenario, for the value in our dictionary, we pass in a tuple containing the dataframe, its index column, and its time index. More information about the possible parameters can be found in the DFS documentation.\n",
```

Comment:
Can you link to the actual page here? Sphinx should have a way to do that in Jupyter notebooks